# spatial-omics
Spatial-omics are all you need

## Voting for sptial method!

[Click here to vote!](https://strawpoll.com/6QnMQV81Vne)




## SRT tools

- Inference based
  
scCube:
[Simulating multiple variability in spatially resolved transcriptomics with scCube](https://www.nature.com/articles/s41467-024-49445-0)


SPIAT:
[Spatial analysis with SPIAT and spaSim to characterize and simulate tissue microenvironments](https://www.nature.com/articles/s41467-023-37822-0)

scDesign3:
[scDesign3 generates realistic in silico data for multimodal single-cell and spatial omics](https://www.nature.com/articles/s41587-023-01772-1)


SRTsim:
[SRTsim: spatial pattern preserving simulations for spatially resolved transcriptomics](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-02879-z)

Spider:
[Spider: a flexible and unified framework for simulating spatial transcriptomics data](https://www.biorxiv.org/content/10.1101/2023.05.21.541605v1.full.pdf)


SPACEL: 
[SPACEL: deep learning-based characterization of spatial transcriptome architectures](https://www.nature.com/articles/s41467-023-43220-3)

Spacia:
[Mapping cellular interactions from spatially resolved transcriptomics data](https://www.nature.com/articles/s41592-024-02408-1?fromPaywallRec=false)

COMMOT:
[Screening cell–cell communication in spatial transcriptomics via collective optimal transport](https://www.nature.com/articles/s41592-022-01728-4)

CellCharter:
[CellCharter reveals spatial cell niches associated with tissue remodeling and cell plasticity](https://www.nature.com/articles/s41588-023-01588-4)


- Spatial resolved

stLearn:
[Robust mapping of spatiotemporal trajectories and cell–cell interactions in healthy and diseased tissues](https://www.nature.com/articles/s41467-023-43120-6?fromPaywallRec=false)

SPATA2:
[Inferring histology-associated gene expression gradients in spatial transcriptomic studies](https://www.nature.com/articles/s41467-024-50904-x?fromPaywallRec=false#Abs1)

Bayespace:
[Spatial transcriptomics at subspot resolution with BayesSpace](https://www.nature.com/articles/s41587-021-00935-2?fromPaywallRec=false)

Squidpy:
[Squidpy: a scalable framework for spatial omics analysis](https://www.nature.com/articles/s41592-021-01358-2)

STADIA:
[Statistical batch-aware embedded integration, dimension reduction, and alignment for spatial transcriptomics](https://academic.oup.com/bioinformatics/article/40/10/btae611/7821185)


imcRtools:
[Tools for IMC data analysis](https://bodenmillergroup.github.io/imcRtools/articles/imcRtools.html)


IMC analysis:
[Analysis workflow for IMC data](https://bodenmillergroup.github.io/IMCDataAnalysis/)


spicyR:
[Spatial Linear and Mixed-Effects Modelling with spicyR](https://www.bioconductor.org/packages/release/bioc/vignettes/spicyR/inst/doc/spicyR.html)

lisaClust:
[Introduction to Clustering of Local Indicators of Spatial Assocation (LISA) curves](https://www.bioconductor.org/packages/release/bioc/vignettes/lisaClust/inst/doc/lisaClust.html)



SpaGCN:
[SpaGCN: Integrating gene expression, spatial location and histology to identify spatial domains and spatially variable genes by graph convolutional network](https://www.nature.com/articles/s41592-021-01255-8)


CRAWDAD:
[Characterizing cell-type spatial relationships across length scales in spatially resolved omics data](https://www.nature.com/articles/s41467-024-55700-1)


STASCAN:
[STASCAN deciphers fine-resolution cell distribution maps in spatial transcriptomics by deep learning](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-024-03421-5)

GraphST:
[Spatially informed clustering, integration, and deconvolution of spatial transcriptomics with GraphST](https://www.nature.com/articles/s41467-023-36796-3)

stSME:
[Robust mapping of spatiotemporal trajectories and cell–cell interactions in healthy and diseased tissues](https://www.nature.com/articles/s41467-023-43120-6)

MuSpAn:
[MuSpAn: A Toolbox for Multiscale Spatial Analysis](https://www.biorxiv.org/content/10.1101/2024.12.06.627195v2)

GSDensity:
[Pathway centric analysis for single-cell RNA-seq and spatial transcriptomics data with GSDensity](https://www.nature.com/articles/s41467-023-44206-x)


## SRT related studies

[Computational Approaches and Challenges in Spatial Transcriptomics](https://academic.oup.com/gpb/article/21/1/24/7274161)


[Giotto: a toolbox for integrative analysis and visualization of spatial expression data](https://genomebiology.biomedcentral.com/counter/pdf/10.1186/s13059-021-02286-2.pdf)


[Systematic evaluation with practical guidelines for single-cell and spatially resolved transcriptomics data simulation under multiple scenarios](https://genomebiology.biomedcentral.com/counter/pdf/10.1186/s13059-024-03290-y.pdf)


[Inferring super-resolution tissue architecture by integrating spatial transcriptomics with histology](https://www.nature.com/articles/s41587-023-02019-9)


[Spatial cellular architecture predicts prognosis in glioblastoma](https://www.nature.com/articles/s41467-023-39933-0)


[The dawn of spatial omics](https://pmc.ncbi.nlm.nih.gov/articles/PMC7614974/)


[Spatial omics and multiplexed imaging to explore cancer biology](https://www.nature.com/articles/s41592-021-01203-6)


[Methods and applications for single-cell and spatial multi-omics](https://www.nature.com/articles/s41576-023-00580-2)


[Spatial transcriptomics in glioblastoma: is knowing the right zip code the key to the next therapeutic breakthrough?](https://pmc.ncbi.nlm.nih.gov/articles/PMC10618006/)


[STASCAN deciphers fine-resolution cell distribution maps in spatial transcriptomics by deep learning](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-024-03421-5)


[SpottedPy quantifies relationships between spatial transcriptomic hotspots and uncovers environmental cues of epithelial-mesenchymal plasticity in breast cancer](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-024-03428-y)

## scRNA

[Comparison and evaluation of statistical error models for scRNA-seq](https://link.springer.com/article/10.1186/s13059-021-02584-9)


[Directing cellular transitions on gene graph-enhanced cell state manifold](https://www.biorxiv.org/content/10.1101/2024.10.27.620174v1)


## Other collections/Refs

[Spatial-omics are all you need](https://github.com/jligm-hash/spatial-omics)

[awesome spatial omics](https://github.com/crazyhottommy/awesome_spatial_omics)


[scRNA-tools](https://www.scrna-tools.org/)


