# spatial-omics
Spatial-omics are all you need

## SRT tools

scCube:
[Simulating multiple variability in spatially resolved transcriptomics with scCube](https://www.nature.com/articles/s41467-024-49445-0)


SPIAT:
[Spatial analysis with SPIAT and spaSim to characterize and simulate tissue microenvironments](https://www.nature.com/articles/s41467-023-37822-0)

scDesign3:
[scDesign3 generates realistic in silico data for multimodal single-cell and spatial omics](https://www.nature.com/articles/s41587-023-01772-1)


SRTsim:
[SRTsim: spatial pattern preserving simulations for spatially resolved transcriptomics](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-02879-z)


## SRT related studies

[Computational Approaches and Challenges in Spatial Transcriptomics](https://pmc.ncbi.nlm.nih.gov/articles/PMC10372921/)


[Giotto: a toolbox for integrative analysis and visualization of spatial expression data](https://genomebiology.biomedcentral.com/counter/pdf/10.1186/s13059-021-02286-2.pdf)


[Systematic evaluation with practical guidelines for single-cell and spatially resolved transcriptomics data simulation under multiple scenarios](https://genomebiology.biomedcentral.com/counter/pdf/10.1186/s13059-024-03290-y.pdf)


[Inferring super-resolution tissue architecture by integrating spatial transcriptomics with histology](https://www.nature.com/articles/s41587-023-02019-9)


[Spatial cellular architecture predicts prognosis in glioblastoma](https://www.nature.com/articles/s41467-023-39933-0)

